tests/features/contact.feature

```diff
@@ -1,20 +0,0 @@
-@lightning @core @contact-form @api
```

Comment:
Both of these are testing post-install configuration functionally...which is exactly what ConfigIntegrityTest does. Therefore, I moved these into there.